save-demo-agent/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+
+
+@Suppress("DSL_SCOPE_VIOLATION", "RUN_IN_SCRIPT")  // https://github.com/gradle/gradle/issues/22797
+plugins {
+    kotlin("multiplatform")
+    alias(libs.plugins.kotlin.plugin.serialization)
+    id("com.saveourtool.save.buildutils.code-quality-convention")
+}
+
+kotlin {
+    val nativeTarget = when (System.getProperty("os.name")) {
```

Comment:
Can you use Gradle's `DefaultNativePlatform` API instead, please?

Reply:
Outdated, was changed by nulls in his save-demo-agent grade fix